drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -117,6 +117,10 @@ static void copy_pvt(struct gps_pvt *dest, nrf_gnss_pvt_data_frame_t *src)
 		dest->sv[i].elevation = src->sv[i].elevation;
 		dest->sv[i].azimuth = src->sv[i].azimuth;
 		dest->sv[i].signal = src->sv[i].signal;
+		dest->sv[i].in_fix =
+			src->sv[i].flags == NRF_GNSS_SV_FLAG_USED_IN_FIX;
+		dest->sv[i].unhealthy =
+			src->sv[i].flags == NRF_GNSS_SV_FLAG_UNHEALTHY;
```

Comment:
I would still propose to do it like this:

```
dest->sv[i].in_fix =
	(src->sv[i].flags & NRF_GNSS_SV_FLAG_USED_IN_FIX) ==
		NRF_GNSS_SV_FLAG_USED_IN_FIX;
```
The approach you used would work if only one of the flags would be set at a time, but we shouldn't make that kind of assumption. Even with the current MFW 1.2.2 bit 0 "SV is being tracked" seems to be used even though it has been deprecated in the GNSS RPC API, so that bit is almost always set for SVs.

Reply:
Right, right, don't know what I was thinking..
Hopefully okay now

Reply:
Looks good now!